.travis.yml

```diff
@@ -10,3 +10,10 @@ install:
 
 before_script:
   - "flake8 ."
+
+branches:
+  only:
+    - master
+
+virtualenv:
+  system_site_packages: true
```

Comment:
So that is would use virtualenv 
![noenv](https://cloud.githubusercontent.com/assets/16641417/13484502/697d710a-e139-11e5-995c-c75e302fc14e.jpg)